website/versioned_docs/version-0.7.0/03-reference/03-sql/01-ddl/06-view/_category_.yml

```diff
@@ -0,0 +1,4 @@
+label: 'Logical View'
```

Comment:
I think Version 0.7.0 is a released version and should not include the view.

Reply:
Yes, please don't put the new doc into the `version-0.7.0` folder, it's immutable after tagged.
I will tagging the new version this week with the command `yarn run docusaurus docs:version 0.7.[x]`, and your new documents will automatic in it soon.

Reply:
get it